dependencies/pom.xml

```diff
@@ -1101,6 +1104,17 @@
                 <artifactId>neo4j-java-driver</artifactId>
                 <version>${version.lib.neo4j}</version>
             </dependency>
+            <!-- Bean Validation -->
+            <dependency>
+                <groupId>javax.validation</groupId>
+                <artifactId>validation-api</artifactId>
+                <version>${version.lib.validation-api}</version>
+            </dependency>
```

Comment:
related to my previous comment. Jakarta bean validation has been referenced only in the helidon module, but not here (where it should be :-) )

Reply:
Switched to `jakarta`

Reply:
Why did you remove the API? :-)